contracts/SpokePool.sol

```diff
@@ -1223,24 +1223,28 @@ abstract contract SpokePool is
             // need to unwrap it to native token before sending to the user.
             if (!relayExecution.slowFill)
                 IERC20Upgradeable(relayData.destinationToken).safeTransferFrom(msg.sender, address(this), amountToSend);
-            _unwrapwrappedNativeTokenTo(payable(relayData.recipient), amountToSend);
+            _unwrapwrappedNativeTokenTo(payable(relayExecution.updatedRecipient), amountToSend);
             // Else, this is a normal ERC20 token. Send to recipient.
         } else {
             // Note: Similar to note above, send token directly from the contract to the user in the slow relay case.
             if (!relayExecution.slowFill)
                 IERC20Upgradeable(relayData.destinationToken).safeTransferFrom(
                     msg.sender,
-                    relayData.recipient,
+                    relayExecution.updatedRecipient,
+                    amountToSend
+                );
+            else
+                IERC20Upgradeable(relayData.destinationToken).safeTransfer(
+                    relayExecution.updatedRecipient,
                     amountToSend
                 );
```

Comment:
Nit: should we add the `tsconfig` setting to require brackets after conditionals?

Reply:
do you know which config we should set for that? I marginally favor requiring brackets but not a big deal